R/readWQPdata.R

```diff
@@ -96,12 +96,25 @@
 #' startDate <- as.Date("2013-01-01")
 #' nutrientDaneCounty <- readWQPdata(countycode="US:55:025",startDate=startDate,
 #'                         characteristicType="Nutrient")
+#' secchi.names = c("Depth, Secchi disk depth", 
+#'                  "Depth, Secchi disk depth (choice list)", 
+#'                  "Secchi Reading Condition (choice list)", 
+#'                  "Secchi depth", 
+#'                  "Water transparency, Secchi disc")
+#' args <- list('startDateLo' = startDate, 
+#'              'startDateHi' = "2013-12-31", 
+#'               statecode="WI", 
+#'               characteristicName=secchi.names)
 #' 
-#'                         
+#' wqp.data <- readWQPdata(args)                  
 #' }
 readWQPdata <- function(..., zip=FALSE, querySummary=FALSE){
```

Comment:
So in a comment I have "TODO:" and it calls out extracting the defaults (`zip` for WQP, `convertType` and something else for NWIS), meaning if this PR was a good start, it would probably comply with your last "main use-case" statement when that TODO was done. 

I definitely had not considered multiple lists, and did assume it was going to be all-or-none (except for the default non-`...` args)....I'd need to wrap my head a bit around how to deal with that.